### PR TITLE
Creators Landing Page

### PR DESCRIPTION
**Pull request by rossmoody**
Refers: https://github.com/brave-intl/publishers/issues/1592

Preview build: https://bat-publishers-staging-2.herokuapp.com/

None of the actual input/button functionality is wired up in any way. I used my best judgement for the hrefs but they can be[ reviewed in locale.](https://github.com/brave-intl/publishers/blob/2cbf0d882181cfca344e3b6e95d6480ddf7ff585/public/creators-landing/src/locale/en.js)

## Unplanned additions

1. Sign up and sign in pages were added to the mix of this PR. Though not initially in the scope, it bugged me how different they looked and we had the style foundation setup. So my proposal for this would be:

- Old: `http://publishers.basicattentiontoken.org` AND `https://brave.com/creators`
New: `new Creators landing page / creators.brave.com`
- Old: `https://publishers.basicattentiontoken.org/publishers/sign_up`
New: `new Creators Sign Up page / creators.brave.com/sign-up`
- Old: `https://publishers.basicattentiontoken.org/publishers/log_in`
New: `new Creators Sign In page / creators.brave.com/sign-in`

2. The gradient background animates. Cause why not.

### CODE REVIEW

**Comment by corymcdonald** [comment]
Ross, this looks great! 

Some thoughts in no particular order

- Now that we have are going to Reactify the sign up and sign in pages, we'll likely want to figure out a good place to show an error message on those screens.

- Have you discussed with David or product about the renaming of the URL? I don't think it will be such a trivial change. 

- One alternative to setting up A/B testing, is to just simply look at the current matomo statistics on the funnel and then compare / contrast. If we do intend to make this non-passive with the `creators.brave.com` domain replacement then we'll need to make another issue to discussing everything that could go into that.

- Have you thought about SEO with this? I'm not quite familiar with any tactics or anything but we might want to consider that. Ideally if someone searched "Earn money with youtube" then creators website would pop up.

File: public/creators-landing/src/components/icons/SwipeRight.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+export const SwipeRightIcon = () => (
```
Comment (corymcdonald) [inline public/creators-landing/src/components/icons/SwipeRight.js]: i'd swipe right on this re-design 👉😄👉

File: public/creators-landing/src/locale/en.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+      },
+      btn: {
+        signup: 'Sign up now'
+      }
+    },
+    signin: {
+      heading: 'Sign in to your account',
+      subhead:
+        'You will receive an email momentarily with a secure login link.',
+      input_placeholder: 'Enter your email',
```
Comment (corymcdonald) [inline public/creators-landing/src/locale/en.js]: Usually javascript uses `camelCase` for variable names

File: public/creators-landing/src/sections/main-section/index.js [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+          </Box>
+          <Box direction='row' pad={{ vertical: '24px' }} width='100%'>
+            <PrimaryButton
+              label={locale.main.home.btn.signup}
+              a11yTitle='Sign up'
+            />
+          </Box>
+        </Box>
+      </Container>
+      <SwoopBottom />
```
Comment (corymcdonald) [inline public/creators-landing/src/sections/main-section/index.js]: I was having problems where the swoop bottom was on top of the "Sign Up" button so I couldn't click on the bottom half when my web browser was like 75% full screen. Can you investigate?

File: public/creators-landing/src/sections/main-section/index.js [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+        <Box
+          direction='column'
+          pad='large'
+          margin={{ top: '80px', bottom: '60px' }}
+        >
+          <Heading
+            level='1'
+            color='white'
+            margin={{ vertical: 'small' }}
+            a11yTitle='Headline'
```
[the remark this answers is not in the document]
Comment (rossmoody) [reply public/creators-landing/src/sections/main-section/index.js]: got carried away with the a11y title. no idea why. omitting.

File: public/creators-landing/src/locale/en.js (hunk 2 above)
Comment (rossmoody) [reply public/creators-landing/src/locale/en.js]: fixed

File: public/creators-landing/src/sections/main-section/index.js (hunk 3 above)
Comment (rossmoody) [reply public/creators-landing/src/sections/main-section/index.js]: great spot. swoop was on top of bottom 1/6 of button. pushed down.

**Comment by rossmoody** [comment]
After meeting as a team, it was decided to move the landing page to brave.com/creators via Wordpress and punt any subdomain change until a top level domain name can be agreed upon at a future date.

**Comment by corymcdonald** [comment]
need to make the built with bat a link

**Comment by rossmoody** [comment]
@dlipeles Done! though right as I was committing I realized, the transition might not be quite as smooth as hoped cause doesn't login happen via link from email that generates a new tab?

**Comment by dlipeles** [comment]
@rossmoody Thanks, and yes you are right, the hope would be to move to a more traditional password-based sign in, in the future. 

Then no need to jump from app to e-mail client, back to app ⛳️

**Pull request merged**